index.html

```diff
@@ -107,23 +106,23 @@ <h3>Multi-Post Storiese</h3>
               ever since the 1500s, when an unknown printer took a standard
               dummy text.
             </p>
-            <div id="tb3">
-              <span id="a">HTML</span>
-              <span id="b">CSS</span>
-              <span id="c">BOOTSTRAP</span>
-              <span id="d">RUBY</span>
+            <div id="tb1">
+              <span id="a1">HTML</span>
+              <span id="b1">CSS</span>
+              <span id="c1">BOOTSTRAP</span>
+              <span id="d1">RUBY</span>
             </div>
```

Comment:
- [ ] `required`
- [ ] Great job so far 👏 
- [ ] However the list of project technologies are supposed to `ul` and each of them in a `li` see example `below`
![image](https://user-images.githubusercontent.com/66230306/186690799-18f92099-0f01-46bc-8bf3-46ea9828594f.png)
Kindly make amends on all the technologies